<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawDataView</class>
 <widget class="QWidget" name="RawDataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="locationIdLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QHexEdit" name="hexEditor" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="saveRamPushButton">
         <property name="text">
          <string>Save To Ram</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveFlashPushButton">
         <property name="text">
          <string>Save To Flash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadRamPushButton">
         <property name="text">
          <string>Load From Ram</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadFlashPushButton">
         <property name="text">
          <string>Load From Flash</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>qhexedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
